test/function/samples/warn-for-load-and-transform/_config.js

```diff
@@ -0,0 +1,43 @@
+module.exports = defineTest({
```

Comment:
Could you move this test into the `/deprecations` subfolder? This is where all other tests for deprecation warnings are located. This makes it easier to go through these when doing major releases.

Reply:
Alright, this is indeed a very suitable spot.